docs/public/_redirects

```diff
@@ -378,18 +378,28 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /:lang/joy-ui/core-features/global-variant/ /:langjoy-ui/main-features/global-variants/ 301
 /joy-ui/core-features/automatic-adjustment/ /joy-ui/main-features/automatic-adjustment/ 301
 /:lang/joy-ui/core-features/automatic-adjustment/ /:lang/joy-ui/main-features/automatic-adjustment/ 301
-/joy-ui/core-features/perfect-dark-mode/ /joy-ui/main-features/perfect-dark-mode/ 301
-/:lang/joy-ui/core-features/perfect-dark-mode/ /:lang/joy-ui/main-features/perfect-dark-mode/ 301
```

Comment:
This is a double redirection. it was added in https://github.com/mui/material-ui/pull/34613/files#diff-7a0cbac74402caf0e99ba338a9e7c42684580a1f85e149110c3c2ca6e38055afR390 @samuelsycamore A tip, when adding a new redirection, it's a good practice to also update the redirections that might already point to the new "old URL". It avoids a double 301.